mlflow/tracking/client.py

```diff
@@ -189,6 +353,31 @@ def delete_experiment(self, experiment_id):
         Delete an experiment from the backend store.
 
         :param experiment_id: The experiment ID returned from ``create_experiment``.
+
+        .. code-block:: python
+            :caption: Example
+
+            from mlflow.tracking import MlflowClient
+
+            # Create an experiment name, which must be unique and case sensitive
```

Comment:
This sounds a bit strange to me because what we create here is an experiment, not a name.

Reply:
As you know experiment names are case sensitive. We create an experiment that has a name, which a) must be unique b) names are case sensitive. An experiment name with `E` is different than one with `e.`

Create an experiment, with a name that is unique and case sensitive.

  ```
   def print_experiement_info(experiment):
        print("Name:{}".format(experiment.name))
        print("Experiment_id:{}".format(experiment.experiment_id))
        print("Lifecycle_stage:{}".format(experiment.lifecycle_stage))

    # Create experiments. Use names which are unique and case sensitive
    for e in ["E", "e"]:
        experiment_id = mlflow.create_experiment(e)
        experiment = mlflow.get_experiment(experiment_id)
        print_experiement_info(experiment)
```

```
Name:E
Experiment_id:1
Lifecycle_stage:active
Name:e
Experiment_id:2
Lifecycle_stage:active
```